src/Localizers.jl

```diff
@@ -239,6 +269,115 @@ function Localizer(localization::SECFisher, p::IT, d::IT, J::IT, T = Float64) wh
     return Localizer{SECFisher, T}((cov) -> sec_fisher(cov, J))
 end
 
+"""
+For `N_ens >= 6`: The sampling distribution of a correlation coefficient for Gaussian random variables is, under the Fisher transformation, approximately Gaussian. To estimate the standard deviation in the sampling distribution of the correlation coefficient, we draw samples from a Gaussian, apply the inverse Fisher transformation to them, and estimate an empirical standard deviation from the transformed samples.
+For `N_ens < 6`: Approximate the standard deviation of correlation coefficient empirically by sampling between two correlated Gaussians of known coefficient. Likely will be a poor approximation even with large numbers of samples due to Gaussian approximation.
```

Comment:
I don't think we should say "Likely will be a poor approximation even with large numbers of samples due to Gaussian approximation", because the Fisher transformation method also makes a Gaussian assumption. We need to assume the distribution of the random variables in order to derive the sampling distribution of the correlation coefficient.

Reply:
I think both are a poor approximation at this low a number without the distributional knowledge. I will modify the statement

Reply:
On second thought I've just removed it.